updater/internal/updater/updater.go

```diff
@@ -145,3 +151,33 @@ func (u *Updater) removeTmpDir() {
 		u.logger.Error("failed to remove temporary directory", zap.Error(err))
 	}
 }
+
+// cleanup removes unnecessary files now that agent has updated from v1 to v2.
+// Update is already successfully completed at this point, so we don't want to return errors.
+func (u *Updater) cleanup() {
+	files := []string{filepath.Join("log", "collector.log"), "config.yaml", "manager.yaml", "updater", "config.bak.yaml", "logging.yaml", "package_statuses.json"}
```

Comment:
These are the files we delete once a v1 to v2 update is successful, let me know if you'd rather keep any of these around instead of deleting. Conversely, I currently keep around `log/updater.log` so if you want that deleted instead let me know